components/EmailValidator.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="d-inline-block w-100">
+    <validating-form
+      :class="{ 'justify-content-around': center, 'd-flex': center }"
```

Comment:
d-flex: center? Should that just be d-flex to set display:flex and justify to centre?

Reply:
Yes.